ttnn/cpp/pybind11/pytensor.cpp

```diff
@@ -386,17 +402,38 @@ Tensor convert_python_tensor_to_tt_tensor(
         TT_THROW("The argument must be of type torch.Tensor or numpy.ndarray!");
     }
 
+    // TODO: Remove check of num_elements from python against volume of ttnn::SimpleShape
```

Comment:
why remove? looks like a healthy check to me

Reply:
You get shape and num_elements directly from python tensor. I highly doubt it can mismatch, but we can keep as a check for external libraries